src/F0_output.py

```diff
@@ -74,14 +112,18 @@ def evaluate_dict(dict_values):
         plots.capacities(
             dict_values["simulation_settings"],
             dict_values["project_data"],
-            dict_values["kpi"]["cost_matrix"]["label"],
+            dict_values["kpi"]["scalar_matrix"]["label"],
             dict_values["kpi"]["scalar_matrix"]["optimizedAddCap"],
         )
+    return
 
-    # plot annuity, first-investment and om costs
-    plots.costs(dict_values)
 
-    # Write everything to file with multipe tabs
+def store_scalars_to_excel(dict_values):
+    """
+    All output data that is a scalar is storage to an excellent file tab. This could for example be economical data or technical data.
+    :param dict_values: dict Of all input and output parameters up to F0
```

Comment:
```suggestion
    :param dict_values: dict of all input and output parameters up to F0
```
At this point it might be faster to use search and replace for this typo :)